pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -219,6 +251,29 @@ def submissions(self):
                                        creation_time=self.object.commented_on)
                               .order_by("id"))
 
+    @cached_property
```

Comment:
hi @phlax, it should be cached because we call it twice otherwise. but probs I misunderstood.